tensorflow_graphics/projects/mesh_rcnn/ops/cubify.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Implementation of the cubify operation for Mesh R-CNN."""
+
+import tensorflow as tf
+
+from tensorflow_graphics.projects.mesh_rcnn.structures.mesh import Meshes
+from tensorflow_graphics.util import shape
+
+
+def _ravel_index(index, dims):
+  """
+  Computes a linear index in an array of shape dims.
+  Reverse functionality of tf.unravel_index.
+  Args:
+    index: A Tensor with each row corresponding to indices into an array
+    of dimension dims.
+    dims: The shape of the array to be indexed.
```

Comment:
index: An `int` tensor of shape `[N,3]`, where each row corresponds to an index into an array of dimension `dims`.
dims: An `int` tensor of shape `[3,]`  denoting shape of array to be indexed.

Also fix the type and shape in args for all methods.